### DISCUSSION TITLE
Engram vs Atreus Keyboard

### DISCUSSION THREAD

**Discussion by Tristen-Sinanju** [binarybottle/engram-layout#20] [Show and tell]
Firstly, thanks for all the research and work...before switching to engram 100%, i was about 110wpm in dvorak... I previously tried workman, then mtgap, before settling on a slightly modified engram... the goal was to get rid of the center column. (speed is nice, but i'm ergonomics focused. )

I really enjoy the consideration put into ease of reaching vs curling, and the placement of ou ld (though my stint with mtgap gets me in trouble a lot still, hahaha) ... but to that end, i like that wy were brought in, compared to mtgap, because they feel a lot more frequent than their replacements vb ... which i alt-finger with my ring fingers to avoid wrist rotation. 

i enjoy the combination of alternation and comfortable rolls, probably since i come from a long history of dvorak. 

as far as how the layout feels, the best way i have to describe it... it would be dvorak + mtgap = engram

The slight modification is only re-pairing the symbols/numbers/punctuation. Again, thank you for the great work. 

![image](https://user-images.githubusercontent.com/84834000/119616827-adae7d00-bdc6-11eb-9354-2621f3ef50f2.png)
![image](https://user-images.githubusercontent.com/84834000/119616912-cc147880-bdc6-11eb-9ca2-3e87bdc7650a.png)
![image](https://user-images.githubusercontent.com/84834000/119617003-efd7be80-bdc6-11eb-9d8a-99a56653d471.png)

**Comment by binarybottle** [comment]
Thank you for describing your experience with other layouts and your first impressions of the Engram layout, @Tristen-Sinanju! I hope you've found a good pairing of symbols and numbers for your particular needs. In an earlier attempt at the layout, I included three characters per key in the two middle columns, but I found issues with compatibility and dissemination across keyboards and when using the keyman app, so I stuck to two characters per key.

**Reply by Tristen-Sinanju** [reply]
very understandable. it always takes a bit of jiggering to get layouts onto 40%'s anyway. 

the punctuation works out pretty well, but without reflashing from chrysalis to something like qmk, i can't separate the standard pairings... this doesn't bother me, though, since it was super easy to work around.... so i thought i'd share. 

it's been a fantastic layout so far. very comfortable, and hasn't been too difficult to learn.

**Comment by Tristen-Sinanju** [comment]
work on learning engram has been going smoothly... just thought i'd share some slight modifications i made that helped with comfort... and how i was able to fit it into the atreus. cheers

![image](https://user-images.githubusercontent.com/84834000/125207151-3cc90480-e250-11eb-9050-ce2817f88cac.png)
![image](https://user-images.githubusercontent.com/84834000/125207181-7d288280-e250-11eb-896d-5afc424a564b.png)

**Comment by binarybottle** [comment]
Thanks for sharing!  

This is my Ergodox layout, similar to my Kinesis Advantage Pro layout:
https://configure.zsa.io/ergodox-ez/layouts/APXBR/latest/0
If you search for "engram". you'll see others' variations.

**Reply by Tristen-Sinanju** [reply]
[engram-keymap.txt](https://github.com/binarybottle/engram/files/8794378/engram-keymap.txt)

I picked up a cute little lunakey mini. here's the qmk keymap i wrote for it. cheers; i'm still enjoying the layout.

**Reply by binarybottle** [reply]
Wonderful!

**Comment by Tristen-Sinanju** [comment]
Just wanted to check back in. I'm cruising through the 1 minute tests in the high 90's and bursting past 120 on occasion... An absolute win, in my opinion, considering pain had kept me in the 80wpm range for the past few years.

'jo' same-finger-skip-bigram still bothers me... But if that's my biggest complaint... No big deal.

Anyway. Still enjoying engram, and I thought it'd be worth offering an update. Cheers.

**Reply by binarybottle** [reply]
This is so heartening to hear!  I'm so happy that you are typing efficiently with less pain.  

I am curious -- the letter 'j' is one of the least frequently used letters in English.  Are there particular words with 'jo' that you use very frequently, like 'joke', 'join', or 'jovial'?

**Reply by Tristen-Sinanju** [reply]
jobs, journals, and joins, mostly, and sometimes joyfully. admittedly it's more of a side-effect of the linux language than the english language, so the keyboard wasn't originally optimized for it anyway.. and there's a rather high frequency of the word "job" in The Metamorphosis (which i was copying on typelit.io the other day), so it was more noticeable.

**Comment by binarybottle** [comment]
Please see https://github.com/binarybottle/engram-2021/discussions/86#discussion-10126586 for an announcement of the new Engram approach/layouts/repo!
